tests/HardwareTestHarness.md

```diff
@@ -121,61 +103,9 @@ There are three positions for I2C devices with other pin orders. Each of these p
 to a crossbar switch and so four blobs of solder can configure each of these headers into any
 desired pin order. As far as I can tell, most of the cheap modules use the VCC/GND/SCL/SDA order.
 
-
 Breadboard Area
 ===============
 
 All the pins on each D1 Mini and the A port of the MCP23017 are brought out to a breadboard 
 area. This can be used to solder components and/or wires, or even a header could be soldered
 on to transfer all the signals to a conventional breadboard. 
-
-
-ESP8266 Device 0 Connections
-----------------------------
```

Comment:
The description here is more exact and exceeds the spec in the readme, so it should be kept.
e.g.; D4/5 of DUT 1 is not described at all.
Also in general the setup might evolve in the readme but this document describes a hardware version, which will remain unchanged (else it will be a v2)